### ISSUE TITLE
kafka_exporter not support messages rate and bytes rate.

### ISSUE THREAD

**Issue by chenwumail** [danielqsj/kafka_exporter#61]
@danielqsj I use kafka_exporter + prometheus + grafana monitor and alert 2 kafka cluster, but it not support the rate monitor. I think rate monitor is very important for kafka, do you have any plan ? Thank you.

**Comment by chenwumail** [comment]
for topic:
produce message rate
produce bytes rate

for broker:
in message rate
in bytes rate
out message rate
out bytes rate

for consumer group:
consume message rate
consume bytes rate

for consumer group instance:
consume message rate
consume bytes rate

**Comment by skonmeme** [comment]
You may make appropriate metrics by following queries. (it is hard to make metrics for byte rates)

for topic:
producer message rate: kafka_topic_partition_current_offset

for broker:
in message rate: kafka_topic_partition_leader & kafka_topic_partition_current_offset
out message rate: kafka_topic_partition_leader & kafka_consumergroup_current_offset

for consumer group:
consumer message rate: kafka_consumergroup_current_offset

**Comment by bojleros** [comment]
I do not think that having a topic bandwidth monitoring is more important than monitoring network interfaces on each broker against any possible bottleneck or cluster inbalance. This is a job for a node_exporter.

**Comment by tencentemr** [comment]
> for topic: produce message rate produce bytes rate
> 
> for broker: in message rate in bytes rate out message rate out bytes rate
> 
> for consumer group: consume message rate consume bytes rate
> 
> for consumer group instance: consume message rate consume bytes rate
https://www.zabbix.com/cn/integrations/kafka  zabbix got lots of these kind of metrics.
